_posts/2016-09-13-whats-new.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "August changelog - what's new in Sandstorm"
+author: Asheesh Laroia
+authorUrl: https://github.com/paulproteus
+---
+
+August's most visible change is that when new users join a Sandstorm server, they can skip the
```

Comment:
I had the same reaction to my text, honestly. I just wasn't sure what to do about it. How's the new text?